cloud/app/server/server.js

```diff
@@ -525,7 +525,7 @@ class _robotAgent extends Capability {
       if (process.env.CLICKHOUSE_ENABLED === 'true') {
         log.debug('ClickHouse integration enabled');        
         this.telemetry = new TelemetryService();
-
+        await this.telemetry.init();
         await this.telemetry.sendLogs({
```

Comment:
`awaits` like this in the main code execution path could be dangerous. If for some reason they fail or block, the server may not function. Can we call them async?